.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -52,15 +52,14 @@ jobs:
           new_version="$(grep -i 'current_version = ' .bumpversion.cfg | sed -e 's/.* = //')"
           awk -v NEW_VERSION="$new_version" -v CHANGELOG_MESSAGE="${{ github.event.inputs.changelog-message }}" 'NR==3{print "## " NEW_VERSION "\n" CHANGELOG_MESSAGE "\n"}1' CHANGELOG.md > tmp && mv tmp CHANGELOG.md
       - name: Commit and Push Changes
-        if: success()
```

Comment:
Based on https://docs.github.com/en/actions/learn-github-actions/expressions#success, this is the default condition so no need to specify it here